tests/integration/targets/ntnx_ndb_time_machines_with_multi_clusters/tasks/data_access_management_and_snapshots.yml

```diff
@@ -0,0 +1,245 @@
+---
+- debug:
+    msg: Start testing ntnx_ndb_time_machine_clusters
+
+- name: create data access instance with cluster name and sla name
+  ntnx_ndb_time_machine_clusters:
+    tm_uuid: "{{time_machine.uuid}}"
+    cluster:
+      name: "{{cluster.cluster2.name}}"
+    sla:
+      name: "{{sla.name}}"
+  register: out
+  ignore_errors: true
+
+
+
+- name: check listing status
+  assert:
+    that:
+      - out.response is defined
+      - out.tm_uuid is defined
```

Comment:
@Gevorg-Khachatryan-97, output has been changed to time_machine_uuid right ? @alaa-bish  please check this tests